src/coreclr/nativeaot/Runtime/unix/UnixNativeCodeManager.cpp

```diff
@@ -1248,7 +1248,7 @@ bool UnixNativeCodeManager::GetReturnAddressHijackInfo(MethodInfo *    pMethodIn
     }
 
     *ppvRetAddrLocation = (PTR_PTR_VOID)pRegisterSet->pLR;
-#elif
+#else
```

Comment:
The code is identical between the `#if` and `#else`, except for the register name.

Would it be better to add delete this ifdef and add a helper method method to LoongArch and Arm reg displays to abstract away the difference?

```
inline uintptr_t* GetReturnAddressLocation() { return pLR; }
...
inline uintptr_t* GetReturnAddressLocation() { return pRA; }
```